.github/workflows/linux-eic-shell.yml

```
@@ -283,6 +283,47 @@ jobs:
         path: sim_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4hep.root
         if-no-files-found: error
 
+  eicrecon-benchmarks-plugins:
+    runs-on: ubuntu-latest
+    needs:
+    - build
+    - ddsim-gun
+    strategy:
```

Don't fail on first failure, but continue testing all benchmarks. 
```suggestion
    strategy:
      fail-fast: false
```